package.json

```diff
@@ -41,7 +41,8 @@
     "@types/lodash": "4.14.123",
     "@types/node": "10.14.1",
     "@types/papaparse": "5.2.0",
-    "@types/react": "16.8.16",
+    "@types/react": "17.0.42",
+    "@types/react-dom": "17.0.14",
```

Comment:
Updated to versions used in Grafana.

This fixes an issue with the types for the `Input` component. Typescript was complaining about the `translate` prop being required even though it's actually optional.